README.rst

```diff
@@ -134,138 +140,149 @@ PineTime:
+.. image:: res/screenshots/GameOfLifeApp.png
+   :alt: Game of Life running in the wasp-os simulator
    :width: 179
 
-.. image:: res/StepsApp.png
-   :alt: Step counter application running on the wasp-os simulator
+.. image:: res/screenshots/SnakeApp.png
+   :alt: Snake Game running in the wasp-os simulator
    :width: 179
 
-.. image:: res/LauncherApp.png
-   :alt: Application launcher running on the wasp-os simulator
+Time management apps:
+
+.. image:: res/screenshots/AlarmApp.png
+   :alt: Alarm clock application running in the wasp-os simulator
    :width: 179
 
-.. image:: res/SettingsApp.png
-   :alt: Settings application running on the wasp-os simulator
+.. image:: res/screenshots/StopwatchApp.png
+   :alt: Stop watch application running on the wasp-os simulator
    :width: 179
 
-.. image:: res/SoftwareApp.png
-   :alt: Software selection app running on the wasp-os simulator
+.. image:: res/screenshots/TimerApp.png
+   :alt: Countdown timer application running in the wasp-os simulator
    :width: 179
 
+System apps:
 
-wasp-os also contains a library of additional applications for you to choose.
-These are disabled by default but can be easily enabled using the Software
-application (and the "blank" white screen is a torch application):
+.. image:: res/screenshots/LauncherApp.png
+   :alt: Application launcher running on the wasp-os simulator
+   :width: 179
 
-.. image:: res/SelfTestApp.png
-   :alt: Self test application running a rendering benchmark on the simulator
+.. image:: res/screenshots/SettingsApp.png
+   :alt: Settings application running on the wasp-os simulator
    :width: 179
 
-.. image:: res/TorchApp.png
-   :alt: Torch application running on the wasp-os simulator
+.. image:: res/screenshots/SoftwareApp.png
+   :alt: Software selection app running on the wasp-os simulator
    :width: 179
 
-.. image:: res/ChronoApp.png
-   :alt: Analogue clock application running in the wasp-os simulator
+.. image:: res/screenshots/FacesApp.png
+   :alt: Switch watch faces
    :width: 179
 
-.. image:: res/DualApp.png
-   :alt: An other clock application running in the wasp-os simulator
+
+Other apps: (The "blank" white screenshot is a flashlight app)
+
+.. image:: res/screenshots/BeaconApp.png
+   :alt: Flash the relatively powerful HRS LED repeatedly
    :width: 179
 
-.. image:: res/FiboApp.png
-   :alt: Fibonacci clock application running in the wasp-os simulator
+.. image:: res/screenshots/CalculatorApp.png
+   :alt: Calculator running in the wasp-os simulator
    :width: 179
 
-.. image:: res/HaikuApp.png
-   :alt: Haiku application running in the wasp-os simulator
+.. image:: res/screenshots/DemoApp.png
+   :alt: Simple always-on demo for showing off wasp-os at conferences and shows
    :width: 179
 
-.. image:: res/LifeApp.png
-   :alt: Game of Life running in the wasp-os simulator
+.. image:: res/screenshots/DisaBLEApp.png
+   :alt: Small application for disabling bluetooth to save power and enhance security
```

Comment:
DisaBLE is a bit of a hacky (if we ever figure out how to do this properly we'll move the feature to Settings) but whilst it is with us I think it counts as a system app.

Reply:
Agreed. Moved.